src/ops/info.rs

```diff
@@ -115,11 +114,17 @@ fn query_and_pretty_view(
     let owners = try_list_owners(config, source_ids, package_id.name().as_str())?;
     let mut shell = config.shell();
     let stdout = shell.out();
+    // Only suggest cargo tree command when the package is not a workspace member.
+    // For workspace members, `cargo tree --package <SPEC> --invert` is useless. It only prints itself.
+    let suggest_cargo_tree_command =
+        ws_inspect_info.package_id.is_some() && !ws_inspect_info.is_member;
+    let add_is_member_note = ws_inspect_info.is_member;
```

Comment:
Maybe we should generalize this on the PackageId's SourceId's SourceKind.
- If from default registry, do nothing
- If from non-default registry, specify that
- If from git or path, say so

We likely don't care about the level of detail of whether it is a workspace member or not.

Reply:
Sounds make sense. Thanks!

Reply:
It looks like:
<img width="1347" alt="image" src="https://github.com/hi-rustin/cargo-information/assets/29879298/bee13e45-f436-4e2b-9980-277888de0a62">
<img width="1117" alt="image" src="https://github.com/hi-rustin/cargo-information/assets/29879298/dcfa8f4e-d40f-45ac-ac11-84bbba449121">
![Uploading image.png…]()